addonscript/requirements/local.in

```diff
@@ -0,0 +1,3 @@
+-r test.in
```

Comment:
This appears to essentially be equivalent to `test.in`, since `tox` is already in `test.in`. Do we gain other benefits from this file?

Reply:
Actually, we should probably remove `tox` from `test.in` (`test.in` should contain dependencies used when executing the tests - there's no point in installing tox from within tox).

Reply:
Makes sense. Is `local.{in,txt}` going to be standard across all releng projects?

Reply:
That's the plan, yep!